index.d.ts

```diff
@@ -1,3 +1,36 @@
-export const sync: (input: number) => number
-// sleep [duration] ms, return Promise which resolved 2 * duration
-export const sleep: (duration: number) => Promise<number>
+/// <reference types="node" />
+
+export type ResvgRenderOptions = {
+  path?: string
```

Comment:
Remove the `path` option because we never implemented it, see: https://github.com/yisibl/resvg-js/commit/4f66422ddedc9995b219555edfcb7c61652bbf07